packages/api/test/pin.spec.js

```diff
@@ -667,6 +667,16 @@ describe('Pinning APIs endpoints', () => {
       assert.strictEqual(getEffectivePinStatus(pins), 'queued')
     })
 
+    it('should return "queued" at least 1 pin has Unpinned status', () => {
```

Comment:
Did you mean to say "IF at least one pin...?"

Reply:
good catch!